api-reference/_hidden/PdfCell/horizontalAlign.md

```diff
@@ -5,7 +5,36 @@ type: String
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies the horizontal alignment for the text inside the exported cell.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+<!-- Description goes here -->
+
+The default alignment of the content depends on the [type of data](/api-reference/_hidden/GridBaseColumn/dataType.md '{basewidgetpath}/Configuration/columns/#dataType'). 
```

Comment:
The default alignment of the content depends on the [dataType](/api-reference/_hidden/GridBaseColumn/dataType.md '{basewidgetpath}/Configuration/columns/#dataType').